pkg/cmd/stack/translate_test.go

```diff
@@ -1568,6 +1568,7 @@ func TestGetSvcPublicPorts(t *testing.T) {
 						Ports: []model.Port{
 							{
 								ContainerPort: 80,
+								HostPort:      80,
```

Comment:
Reached this point we always need to have both ports set, because we have translated to a compose syntax so we need to have HostPort field defined